datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -133,6 +150,13 @@ func resourceDatadogSecurityMonitoringDefaultRuleRead(ctx context.Context, d *sc
 
 	d.Set("filter", ruleFilters)
 
+	responseOptions := ruleResponse.GetOptions()
+	ruleOptions := []map[string]interface{}{{
+		"decrease_criticality_based_on_env": responseOptions.GetDecreaseCriticalityBasedOnEnv(),
```

Comment:
Is the option `decrease_criticality_based_on_env` always present on GET? Is it possible that GET call can return no option/empty options in response?

Reply:
Currently it can happen but we'll fix it soon. The default value is false, so using `GetDecreaseCriticalityBasedOnEnv()` should be fine.